ontology/yaml/resources/LIGHTING/entity_types/ABSTRACT.yaml

```diff
@@ -151,4 +151,49 @@ PWM:
   description: "Basic power monitoring "
   is_abstract: true
   uses:
-  - power_sensor  
\ No newline at end of file
+  - power_sensor  
+
+#New abstracts for DE-FRA-NEM32
+CCIEM:
+  description: "CIE light color monitoring"
+  is_abstract: true
+  uses: 
+  - lighting_color_cie_xvalue_sensor
+  - lighting_color_cie_yvalue_sensor
+
+CHSLM:
+  description: "HSL light color monitoring"
+  is_abstract: true
+  uses: 
+  - lighting_color_hsl_saturation_percentage_sensor
```

Comment:
@pisuke do we need hsl called out for saturation and hue values?

Reply:
Yes, it's useful to keep hsl there.